test/suites/clustering_move.sh

```diff
@@ -52,8 +52,12 @@ test_clustering_move() {
   LXD_DIR="${LXD_ONE_DIR}" lxc info c1 | grep -q "Location: node1"
 
   # c1 can be moved within the same cluster group if it has multiple members
+  current_location="$(LXD_DIR="${LXD_ONE_DIR}" lxc query /1.0/instances/c1 | jq -r '.location')"
   LXD_DIR="${LXD_ONE_DIR}" lxc move c1 --target=@default
+  [ "$(LXD_DIR="${LXD_ONE_DIR}" lxc query /1.0/instances/c1 | jq -r ".location == \"$current_location\"")" = "false" ]
```

Comment:
That definitely works but I don't know if you are aware that `jq` can set its return code based on comparison like that. That'd be with `jq -re`. Just a proposition, I'm fine with the existing test too.

Reply:
Wasn't aware of it, thanks for the suggestion, looks much cleaner :) Changed.